examples/3d/elixir_euler_sedov_blast_wave.jl

```diff
@@ -82,6 +82,7 @@ callbacks = CallbackSet(summary_callback, amr_callback, stepsize_callback,
 ###############################################################################
```

Comment:
There is still a `# FIXME Taal restore after Taam sync` above.

Reply:
That's because there was no test for it :cry: Fixed in 55a3a1f1.